modules/javafx.media/src/main/native/jfxmedia/jni/NativeAudioSpectrum.cpp

```diff
@@ -55,6 +55,10 @@ Java_com_sun_media_jfxmediaimpl_NativeAudioSpectrum_nativeSetBands(JNIEnv *env,
 {
     CAudioSpectrum *pSpectrum = (CAudioSpectrum*)jlong_to_ptr(nativeRef);
     CJavaBandsHolder *pHolder = new (std::nothrow) CJavaBandsHolder();
+    if (pHolder == NULL) {
```

Comment:
I see more than one occurrence of (std::nothrow) in the code base where the return value is not checked for NULL (directwrite.cpp, GlassAccessible.cpp, GlassTextRangeProvider.cpp - I did not check beyond *.c* wildcard though, so could be more).

Do we want to add a null check there?

Reply:
If so, it's unrelated to this fix.